(DOCSP-9478): Add mention to Realm Data Models in Document Schema Types Doc (#154)
mongodb/docs-realm: source/mongodb/document-schemas.txt

- :ref:`Numbers <schema-type-numbers>`
- :ref:`Booleans <schema-type-booleans>`

.. note::

>    MongoDB Realm can be used to automatically create :doc:`GraphQL

Reviewer: Nitpick: could you omit "be used to" or is that essential to the explanation? Also, how does a user initiate the automatic creation?

Author: "Be used to" is arguably essential because although the process is considered 'automatic', you do have to click some buttons in the UI to generate the data model

Author: I chose not to mention how to initiate the automatic creation because that will be in the linked docs, though perhaps this should be more clear. What do you think about the phrasing: 

```You can create GraphQL types or Realm Data Models in the MongoDB Realm UI. See the `GraphQL</link>` doc or the `Define a Realm Schema</link>` doc to learn how.```

Reviewer: That's more clear. Before, I wasn't sure whether the links were just explanations of what GraphQL types and Realm data models were or if those pages contained instructions on how to trigger the automatic generation.

Author: Thanks and yup realm schema page will have instructions

>    MongoDB Realm can be used to automatically create :doc:`GraphQL

Reviewer 2: Avoid passive voice: who is doing the using?

```suggestion
   You can use MongoDB Realm to automatically create :doc:`GraphQL
```
or

```suggestion
   You can automatically create :doc:`GraphQL
```
or
```suggestion
   MongoDB Realm can automatically create :doc:`GraphQL
```

Author: resolving since the sentence has been rephrased - no passive voice in the new rendition.
